apps/web/components/webhook/WebhookListItem.tsx

```diff
@@ -41,8 +41,8 @@ export default function WebhookListItem(props: { webhook: TWebhook; onEditWebhoo
                 <span
                   key={ind}
                   className={classNames(
-                    "w-max rounded-sm px-1 text-xs ",
-                    props.webhook.active ? "bg-blue-100 text-blue-700" : "bg-blue-50 text-blue-200"
+                    "w-max rounded-sm px-1 text-xs",
+                    props.webhook.active ? "text-grey-200 bg-gray-200" : "bg-grey-50 text-grey-200"
```

Comment:
Asked Ciarán if this blue was intended and he suggested to go for gray, so I changed it